extra-scientific/maxima/autobuild/beyond

```diff
@@ -1,2 +1,2 @@
 # FIXME: fixed version
-install -Dm755 src/binary-ecl/maxima.fas "$PKGDIR"/usr/lib/ecl-16.1.3/maxima.fas
```

Comment:
Best to use the PKGVER variable.

Reply:
Well this is ECL's version, not Maxima's.

Reply:
The version here is not maxima's version, therefore PKGVER variable is not applicable in this scenario,